### DISCUSSION TITLE
Two unordered lists next to each other?

### DISCUSSION THREAD

**Discussion by Yang-Xijie** [squidfunk/mkdocs-material#3726] [Ideas]
It will be great to render

```
* item1
* item 2
* item 3

- item a
- item b
- item c
```

as two different unordered lists, if different characters are used for each list... 

eg. add a blank line between them (GitHub does like this):

* item1
* item 2
* item 3

- item a
- item b
- item c

However, I noticed that mkdocs-material ignores the line which seperates these two lists. I wonder whether it is a bug or it was designed as is?

Reference: https://squidfunk.github.io/mkdocs-material/reference/lists/?h=list#using-unordered-lists

**Comment by squidfunk** [comment]
This was double-posted: #3731
